scripts/irods/test/test_ibun.py

```diff
@@ -286,3 +286,59 @@ def count_objects_in_collection(collection_path):
             lib.remove_resource(self.admin, compound_resource)
             lib.remove_resource(self.admin, cache_resource)
             lib.remove_resource(self.admin, archive_resource)
+
+    @unittest.skip('Generation of large file causes I/O thrashing... skip for now')
+    def test_ibun__issue_7474(self):
+        try:
+            test_file_small = "ibun_test_file_small"
+            test_file_medium = "ibun_test_file_medium"
+            test_file_large = "ibun_test_file_large"
+            lib.make_arbitrary_file(test_file_small, 1000)
+            lib.make_arbitrary_file(test_file_medium, 100*1024*1024)
+
+            # for this very large file, just making it with truncate as make_arbitrary_file
+            # would take too long and contents='random' would use too much entropy
+            lib.make_file(test_file_large, 12*1024*1024*1024, 'arbitrary')   # make_arbitrary
+
+            tar_path = f'{self.admin.session_collection}/somefile.tar'
+            collection_to_tar = 'tardir'
+            extract_collection = 'extracted'
+            collection_to_tar_fullpath = f'{self.admin.session_collection}/{collection_to_tar}'
+            extract_collection_fullpath = f'{self.admin.session_collection}/{extract_collection}'
+
+            # create directories to tar up and extract the tar to
+            self.admin.assert_icommand(f'imkdir {collection_to_tar_fullpath}')
+            self.admin.assert_icommand(f'imkdir {extract_collection_fullpath}')
+
+            # put files into tar colleciton
+            self.admin.assert_icommand(f'iput {test_file_small} {collection_to_tar_fullpath}')
+            self.admin.assert_icommand(f'iput {test_file_medium} {collection_to_tar_fullpath}')
+            self.admin.assert_icommand(f'iput {test_file_large} {collection_to_tar_fullpath}')
+
+            # create tar file
+            self.admin.assert_icommand(f'ibun -cDtar {tar_path} {collection_to_tar_fullpath}')
+
+            # extract tar file
+            self.admin.assert_icommand(f'ibun -x {tar_path} {extract_collection_fullpath}')
+
+            # verify contents of extracted file into irods
+            self.admin.assert_icommand(f'ils -l {extract_collection_fullpath}/{collection_to_tar}', 'STDOUT_MULTILINE', [test_file_small, test_file_medium, test_file_large])
+
+            # get the files and compare contents
+            self.admin.assert_icommand(f'iget {extract_collection_fullpath}/{collection_to_tar}/{test_file_small} {test_file_small}.downloaded')
+            self.admin.assert_icommand(f'diff {test_file_small} {test_file_small}.downloaded', 'EMPTY')
```

Comment:
Have we used `assert_icommand` with linux commands in other tests?

Reply:
I have.

Reply:
Hmm. Not sure how I feel about that.

Not a blocker for this PR. It's just a little weird to see `assert_icommand` and no icommand be invoked. Seems like it could confuse us later on.

Reply:
Looks like `assert_icommand` simply calls `assert_command`.  I can change that.

Reply:
Up to you.

Reply:
I made the change to use assert_command.  I made sure the test still failed without the new code and passed with the new code (both with the skip removed).

Reply:
Excellent!